source_gen/CHANGELOG.md

```diff
@@ -8,13 +10,16 @@
 - Document deduplication behavior for the output of
   `GeneratorForAnnotation.generateForAnnotatedElement`.
 - Support all the glob quotes.
-- Require `analyzer: ^6.9.0`
-- Require Dart 3.6.0
+- Require `analyzer: '>=7.2.0 <8.0.0'`
+- Support the latest `package:dart_style`
+- Add `generateForAnnotatedDirective` in addition to `generateForAnnotatedElement2`
+  because in the analyzer element model V2 directives are not elements.
```

Comment:
```suggestion
- Add `generateForAnnotatedDirective`, which now must be used instead of `generateForAnnotatedElement` in order to support annotations on directives (imports, exports, parts, etc).
```